docs/evaluate/use-cases-design-patterns.mdx

```diff
@@ -0,0 +1,111 @@
+- Custom CI/CD at [Netflix](https://temporal.io/replay-2023/videos/actor-workflows-reliably-orchestrating-thousands-of-flink-clusters-at)
+
+### AI / ML and Data Engineering
+
+AI and ML developers face challenges in system orchestration, such as managing complex data pipelines and job coordination across GPU resources. 
+Temporal's code-first approach helps build reliable services faster, making it popular among AI companies. For example:
+- Orchestrating video processing at [Descript](https://temporal.io/blog/ai-ml-and-data-engineering-workflows-with-temporal)
+- Automating data pipelines at [Neosync](https://temporal.io/blog/ai-ml-and-data-engineering-workflows-with-temporal)
+
+## General Use Cases
+
+### Human in the Loop
+
+"Human in the Loop" systems require human interaction for certain steps, such as customer onboarding, forms, or invoice approval.
+These are event driven system with humans generating events, and may be challenging to implement due to timing or unreliable connections between the human to the rest of the system.
+They can use schedules and timers to prompt for user input.
+
+**Example**: [Background checks example](https://github.com/temporalio/background-checks).
+
+**Code Sample**: [Candidate acceptance example prompting for a response](https://learn.temporal.io/examples/go/background-checks/candidate-acceptance)
+
+### Polyglot Systems
+
+Temporal supports multiple languages, allowing different teams to work in their preferred languages. 
+Systems are built by people and teams with different language skills.
+An advantage of event-driven systems is they can interact via a specific protocol without having to be written in the same language.
+
+**Example**: [Polyglot example](https://github.com/temporalio/temporal-polyglot). An implementer can wrap existing code, a Typescript UI starting a Java Workflow which calls a Python Activity.
+
+### Long Running Tasks
+
+This use case is particularly relevant for scenarios like shopping cart Workflows in an eCommerce app, where you can handling long-running tasks efficiently without managing state in a separate database.
+It processes one message at a time, ensuring each message is processed only once.
+This approach addresses issues that can arise with long message processing times, which in other systems might cause consumer failover (typically with a default 5-minute message poll timeout) and potentially result in duplicate message processing by multiple consumers.
+Temporal's ability to handle extended task durations makes it well-suited for such scenarios.
+The [heartbeat](/encyclopedia/detecting-activity-failures#activity-heartbeat) feature allows you to know that an activity is still working, providing insight into the progress of long-running processes.
+
+**Example**: [eCommerce example](https://docs.temporal.io/blog/build-an-ecommerce-app-with-temporal-part-1).
+
+**Code Sample**: [Temporal eCommerce](https://github.com/temporalio/temporal-ecommerce)
```

Comment:
Suggestion: Rather than calling out this one code sample, let's point them instead to https://learn.temporal.io/examples/ where we're trying to consolidate a collection of these (and eventually link to Code Exchange from).

Reply:
Amended my first suggestion to point to https://learn.temporal.io/examples/ as well.